test/protocol-tests/src/test/java/software/amazon/awssdk/protocol/tests/timeout/BaseApiCallAttemptTimeoutTest.java

```diff
@@ -19,13 +19,14 @@
 
 import java.time.Duration;
 import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.testutils.retry.RetryableTest;
 import software.amazon.awssdk.utils.Pair;
 
 public abstract class BaseApiCallAttemptTimeoutTest extends BaseTimeoutTest {
 
     protected static final Duration API_CALL_ATTEMPT_TIMEOUT = Duration.ofMillis(100);
     protected static final Duration DELAY_BEFORE_API_CALL_ATTEMPT_TIMEOUT = Duration.ofMillis(50);
```

Comment:
I was not able to reproduce the test pertaining to this timeout since the value here is already low. I don't think lowering it under 50ms is going to solve anything.